Userland/Utilities/dd.cpp

```diff
@@ -74,13 +74,29 @@ static int handle_size_arguments(size_t& numeric_value, const char* argument)
         return -1;
     }
 
+    unsigned suffix_multiplier = 1;
+    switch(value[value.length()-1]) {
+        case 'k':
```

Comment:
Should we update the usage message to mention these new suffixes are supported for sizes?

Reply:
Considering the rest of the parameters don't have "syntax" guidelines and the linux dd man page excludes the supported suffixes, I am struggling to think of the most succinct way of mentioning this.

Reply:
The Linux/GNU Coreutils dd(1) man page says the following about size arguments:
> N and BYTES may be followed by the following multiplicative  suffixes:  c=1,  w=2, b=512,   kB=1000,  K=1024, MB=1000*1000,  M=1024*1024,  xM=M,  GB=1000*1000*1000, G=1024*1024*1024, and so on for T, P, E, Z, Y.  Binary prefixes can be used, too: KiB=K, MiB=M, and so on.
Maybe we could do something similar.

Furthermore, while I agree we should probably only implement `k`, `M` and `G`, I think we should allow both lower- and uppercase versions (so does Linux/GNU Coreutils, and POSIX does not say anything about human-readable suffixes).

Reply:
Thank you for posting this, didn't seem to find that on the first look, I updated the usage message for block sizes. In regards to lowercase, I agree with you completely and made those changes, just wasn't sure if I just wanted to exactly follow the issue since I am new :)